docs/data-tools/spack.md

```diff
@@ -0,0 +1,412 @@
+one another for different use cases -- for example different versions, built
+with different compilers, using MPI or hybrid MPI+OpenMP. Spack is principally
+written in Python but has a component written in Answer Set Programming (ASP)
+which is used to determine the required dependencies for a given package
+installation.
+
+Users are welcome to install Spack themselves in their own directories, but we
+are making an experimental installation tailored for ARCHER2 available
+centrally. This page provides documentation on how to activate and install
+packages using the central installation on ARCHER2. For more in-depth
+information on using Spack itself please see the [developers'
+documentation](https://spack.readthedocs.io/en/latest/).
+
+!!! important
+    As ARCHER2's central Spack installation is still in an experimental stage
+    please be aware that we cannot guarantee that it will work with full
+    functionality and we may not be able to provide support.
+
+## Activating Spack
+
+As it is still in an experimental stage, the Spack module is not made available
+to users by default. The development modules must firstly be enabled by running:
+    
+    auser@ln01:~> module use /work/y07/shared/archer2-lmod/apps/dev
+
+Several modules with `spack` in their name will become visible to you. You
+should simply load the `spack` module:
+
+    auser@ln01:~> module load spack
+
+This configures Spack to place its cache on and install software to a directory
+called `.spack` in your base work directory, e.g. at
+`/work/t01/t01/auser/.spack`.
+
+At this point Spack is available to you via the `spack` command. You can get
+started with `spack help`, reading the [Spack
+documentation](https://spack.readthedocs.io/en/latest/), or by testing a
+package's installation.
+
+## Using Spack on ARCHER2
+
+### Installing software
+
+At its simplest, Spack installs software with the `spack install` command:
+
+    auser@ln01:~> spack install gromacs
+
+This very simple `gromacs` installation specification, or spec, would install
+GROMACS using the default options given by the Spack `gromacs` package. The spec
+can be expanded to include which options you like. For example, the command
+
+    auser@ln01:~> spack install gromacs@2024.2%gcc+mpi
+
+would use the GCC compiler to install an MPI-enabled version of GROMACS version
+2024.2.
+
+!!! tip
+    Spack needs to bootstrap the installation of some extra software in order to
+    function, principally `clingo` which is used to solve the dependencies
+    required for an installation. The first time you ask Spack to concretise a
+    spec into a precise set of requirements, it will take extra time as it
+    downloads this software and extracts it into a local directory for Spack's
+    use.
+
+You can find information about any Spack package and the options
+available to use with the `spack info` command:
+
+    auser@ln01:~> spack info gromacs
+
```

Comment:
Possibly add a callout that points to https://packages.spack.io as an alternative way to access this info?

Reply:
Good idea -- done.